### ISSUE TITLE
Default target_accept not mentioned anywhere in the docs

### ISSUE THREAD

**Issue by ricardoV94** [pymc-devs/pymc#7455]
### Description

...

**Comment by kdotmanoj** [comment]
can you provide more info about this?

**Comment by ricardoV94** [comment]
> can you provide more info about this?
> 

I think it's pretty self explanatory.

**Comment by ricardoV94** [comment]
It's a keyword argument to pm.sample

**Comment by kdotmanoj** [comment]
so the value is 0.8 right, found here:

![image](https://github.com/user-attachments/assets/e02a88d7-3424-4fd5-8b92-48ac7bb8266f)

i am finding it difficult to find where to edit as it is autogenerated from the code.

**Comment by ricardoV94** [comment]
> so the value is 0.8 right, found here:
> 
> ![image](https://github.com/user-attachments/assets/e02a88d7-3424-4fd5-8b92-48ac7bb8266f)
> 
> i am finding it difficult to find where to edit as it is autogenerated from the code.

You're looking at nutpie sample function not pymc

**Comment by cluhmann** [comment]
Is `target_accept` a kwarg to `pm.sample()` or just another `**kwargs`? The argument isn't even [included in the API docs](https://www.pymc.io/projects/docs/en/stable/api/generated/pymc.sample.html), so if it's the former, there's a bigger problem.

I mean, it's detailed  [here](https://www.pymc.io/projects/docs/en/stable/api/generated/classmethods/pymc.step_methods.hmc.NUTS.__init__.html#pymc.step_methods.hmc.NUTS.__init__), but I assumed that the sampling defaults would be specified deeper in the code, possibly allowing for different default for different sampling algorithms.

**Comment by Armavica** [comment]
There are at least a couple places where a default is provided deep in the code:

https://github.com/pymc-devs/pymc/blob/747fda3192a5df90cf76bbce641a9542559548c8/pymc/sampling/mcmc.py#L730

https://github.com/pymc-devs/pymc/blob/747fda3192a5df90cf76bbce641a9542559548c8/pymc/sampling/jax.py#L478

https://github.com/pymc-devs/pymc/blob/747fda3192a5df90cf76bbce641a9542559548c8/pymc/step_methods/hmc/hmc.py#L119

https://github.com/pymc-devs/pymc/blob/747fda3192a5df90cf76bbce641a9542559548c8/pymc/step_methods/hmc/base_hmc.py#L72

**Comment by cluhmann** [comment]
Exactly. But those are each specified separately and, it turns out, in a way that creates different defaults for different algorithms.  So I'm not sure what's missing from the docs.

**Comment by ricardoV94** [comment]
> Exactly. But those are each specified separately and, it turns out, in a way that creates different defaults for different algorithms.  So I'm not sure what's missing from the docs.

I couldn't figure out what the default is as an advanced user. Since it's common advice to increase it (even automatic in case of divergences) it should be easy to find what the default is.

**Comment by cluhmann** [comment]
This is caused by `pm.sample()` being a franken-wrapper that dispatches to a bunch of actual sampling methods, but seems like it does something useful all on its own.  Where would you expect the default value to be documented?

**Comment by ricardoV94** [comment]
pm.sample 🤷‍♂️🫠🥹? Or do you have a better idea?

**Comment by cluhmann** [comment]
But ... it's not an argument to `pm.sample()`. I do have a better idea, but you didn't like it (e.g., deprecate `pm.sample`). :joy:

**Comment by ricardoV94** [comment]
> But ... it's not an argument to `pm.sample()`. I do have a better idea, but you didn't like it (e.g., deprecate `pm.sample`). :joy: 

It is, just hidden inside **kwargs. 

It's actually a special kwarg, the only one we accept both directly and as usual step kwargs

**Comment by ricardoV94** [comment]
https://github.com/pymc-devs/pymc/pull/5622

**Comment by kdotmanoj** [comment]
@ricardoV94  do you suggest mentioning the default values of target_accept https://github.com/pymc-devs/pymc/blob/747fda3192a5df90cf76bbce641a9542559548c8/docs/source/api/samplers.rst[here] ?

**Comment by SanketMeghale** [comment]
I can take this next after #8149. Proposed scope: add a short docs note clarifying where 	arget_accept is used in pm.sample and where default values come from for the sampler paths, so users can quickly find the effective default.
